tools/mo/openvino/tools/mo/front/onnx/loader.py

```diff
@@ -70,6 +76,9 @@ def protobuf2nx(graph: Graph, pb):
             graph.add_node(name, kind='op', op='FakeOutput', pb=outp)
             output_ids.append(name)
 
+    # Preserve outputs order
+    graph.outputs_order = output_ids
+
```

Comment:
what if an user specifies different inputs using truncation through `--input`? In this case we shouldn't track order. The same question is for outputs. For this cases we got warnings during serialization.
please check this case

Reply:
Added functionality to preserve order of user specified input/output.